test/dash/dash_parser_content_protection_unit.js

```diff
@@ -1005,7 +1005,7 @@ describe('DashParser ContentProtection', () => {
         // record count
         1,
         // type
-        ContentProtection.PLAYREADY_RECORD_TYPES.RIGHTS_MANAGEMENT,
+        1,
```

Comment:
Can't you just use `shaka.util.PlayReady.PLAYREADY_RECORD_TYPES.RIGHTS_MANAGEMENT` instead of a constant?